docs/reference/ml/trained-models/apis/put-trained-model-vocabulary.asciidoc

```diff
@@ -45,10 +45,14 @@ include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=model-id]
 (array)
 The model vocabulary. Must not be empty.
 
+`merges`::
+(Optional, array)
+The model merges. Must be provided for RoBERTa and BART style models.
```

Comment:
Tried to find more context for this parameter.  Is this correct?:
```suggestion
The model merge rules, which map text to tokens. Must be provided for RoBERTa and BART style models.
```

Reply:
> The model merge rules, which map text to tokens

This is technically not correct. The merges are just space delimited encoded tokens. Like `fo o` which means that `fo o` can be merged into `foo`. I will attempt to clarify better.